wasm-node/rsc/run_node.js

```diff
@@ -0,0 +1,44 @@
+Error.stackTraceLimit = 99;
+
+import init, { setup, Network, WasmNode, WasmNodeConfig, canonical_network_bootnodes, network_genesis } from "/wasm/wasm_node.js";
+
+// initialize wasm
+await init();
+// setup logging and console panic hook
+await setup();
+
+const response = await fetch('/cfg.json');
+const json = await response.json();
+
+console.log("Received config:", json);
+
+const network = json.network;
+let bootnodes = json.bootnodes
+if (bootnodes.length === 0) {
+    bootnodes = canonical_network_bootnodes(network);
+}
+const genesis = network_genesis(network);
+
+document.getElementById("network_id").value = network;
+document.getElementById("genesis").value = genesis;
+document.getElementById("bootnodes").value = bootnodes.join("\n");
+
+document.getElementById("start").addEventListener("click", async function(ev) {
+    document.getElementById("start").setAttribute("disabled", "disabled");
+
+    const network = Number(document.getElementById("network_id").value);
+    const genesis = document.getElementById("genesis").value;
+    const bootnodes  = document.getElementById("bootnodes").value.split("\n");
+
+    console.log("starting with:", network, bootnodes, genesis);
+
+    const config = new WasmNodeConfig(network, genesis, bootnodes);
+    window.node = await new WasmNode(config);
+
+    setInterval(async function() {
+        document.getElementById("peer_id").innerText = JSON.stringify(await window.node.local_peer_id());
```

Comment:
this could be set once before the interval I think

Reply:
ah you missunderstood me I thought about bringing the `peer_id` out of the loop completely as it doesn't change through the node's lifetime. I'm fine with having other stuff updated after the interval for simplicity but up to you

Reply:
Thanks for clarification, it makes sense now. PeerId indeed isn't changing so we can set once 👍